lorenz96-example.ipynb

```diff
@@ -0,0 +1,642 @@
OMjR5RK/axOmwzJll8gMdlt2eWQbAsOySarWKa9eucb1yGwzL5BkMy8GUTCY9vWa5WCyiXq97PiyHQiGkUimG5R7Q77ETZdiAPbt848YNR16Ljiafz2N4eBjRaLSrx4nFYojH4/8/e/ce3Mh134n+e/AgQPAFEEBjRkMNNXpGlixLtjSSaM8qLl9LsmVF63XWayW+HmeTOFvxZlP36m7Vlp2Ms8pra6tSW+X7yqvijLcc+zp3ryNVJPltSWPPy7KtyJEsOYoHFDVjz6O70Q0SBAigz/2DcyjODIcEgUY/v58qlS2SQxyRA6C//Tvn9+NNEgoFXdeRTCYDcQ2mwrKU0u+lxMrCwgIcx2FY3gDDMgWClBK2bQfihZrcFfRZy2EYG6VwfJQ33KwsA6vnlhmagsEwDM+2mnJ8FIWFrusoFAqB2H5bKpXQbDbRaDT8XkqsqOZeDMuXYlimQFhcXITjOAzLEZTP5wMdllXVOwxhuVQqMSx7QNd1TE5OIp1Ou/L9NE1DrVZDq9Vy5ftR/7zsT1Aul3mThEIhCGOjFHbE9ke1WsXU1NRarxl6A8MyBYJlWQDAsVERVCgUsLS0FNg5s2GqLE9PT8OyLHQ6Hb+XEmlujY1S2Bk5OAzD8Oy5rm6SrKysePJ4RP1ya668GxiW/VGtVllVvgyGZQoE27YBgJXlCFIXpkE9t2yaJtLpNHK5nN9L2VKxWISUMtCV+ijQdd2188oAx0cFxcrKiqezZNVNEl70U5B1Oh1YluV7J2yFYdl7y8vLOH36NMPyZTAsUyCoyjLDcvQEfdZyrVYLzFmtrXB81PBJKV3fkqg6I7PJl7/Ua5CXZ5YB7iigYKvVapBSBiYsZ7NZjI2NMSx7aH5+HgAwOzvr80qCiWGZAoHbsKMr6GHZy4Y/g2JYHj61zd3NvxPZbBZTU1OsLPvM6yMXnLFNYRCksVEKx0d5S4VlVpY3xrBMgWDbNrLZ7NDHeZD3VLOIIG/DDktDC1UBZ1geHnXh6OY2bIDjo4LA68pyLpfD2NgYwzIFmtvd/93AsOytarWKUqmEsbExv5cSSAzLFAiWZbGqHFGZTAajo6OBrCw7jhOos1pbSSaTyOfzDMtDNKwLR03TGJp8pm6EeHlzjOOjKOiCGpZ1XYfjOH4vJRbm5+dZVd4EwzIFgmVZPK8cYYVCIZCVZdu24ThOKDphKxwfNVzqZ+v2lsRKpYJ6vY7l5WVXvy/1zjRNTE5OIpVKefaYHB9FQWcYBqampjx9XmylVCqh2+2uHdGj4anX69B1neeVN8GwTIHAsBxthUIhkJVltaYwheXp6WmG5SHSdR3j4+OuHwnh+VX/eTljWdE0DYZhcNwbBVaQxkYp7IjtHZ5X3hrDMgUCw3K05fN5hmWXFItFmKaJbrfr91IiaVgXjhwf5T8/mvlpmgYpJW9wUWAFscklw7J3qtUqhBDYvXu330sJLIZl8l2r1UKr1WJYjrBCoQDbtgMX8MIYlkulEqSUgdzWHgXDCsscH+UvNRLM61DA8VEUZH49L7YyPT0NIQTDsgeq1Sp27NiBbDbr91ICi2GZfMcZy9GXz+chpYRt234v5QKGYSCdTiOXy/m9lJ6pixpWqtynKoDDuHBMp9MoFAqsLPtkeXkZrVbLl23YAHcUUDDV63V0Op3AbcNOpVLI5/MMy0MmpUS1WuUW7C0wLJPvOGM5+oI6a7lWq62NYwoLzloensXFRbTbbdfHRimVSoWhySd+7SJR59/5e6cgCmInbIXjo4bPNE3U63WG5S0wLJPvWFmOPjWqJWhhOYjbz7aitqcxLLtv2BeOnLXsH69nLCtCCGiaxm3YFEjD6v7vBobl4atWqwDY3GsrDMvkO7U1l2E5uoJaWTZN09OZq25IpVKYmppiWB6CYV84VioVNBoNLC0tDeX70+WpGct+9Cfg+CgKKvW8CGpYrtVqaLfbfi8lsqrVKpLJJGZmZvxeSqAxLJPvLMtCIpHA2NiY30uhIcnlckin04FqSuU4DizLClVzL6VYLDIsD8Gww7I6v8rqsvdM00QikfDlpqymadB1HY7jeP7YRJvRdR25XC6QzZ3UcRgV6Ml98/PzmJmZCdSM7SBiWCbf2baNqampUJ0bpe0RQgRu1rJt23AcJ3TbsAHOWh4WdeE4Ojo6lO/PZk/+MQwD+XweiYT3lz2apqHb7fKinwInyEeROD5quFRzr9nZWb+XEnhDedcQQjCEU884YzkeghaW1VrCtg0bWL2IMAyDlSqXDWtslFIqlSCEYFj2gWEYvu0i4fgoCqphdf93gwrLfN4Mx5kzZ9BsNnleuQc9hVohxI+FEA+u+/ecEOLTQojrNvjaXwbAAwbUM8uy2Ak7BvL5fKC2YfvV8McNxWJxbRs5uWfYF46pVArFYpHbsH1gmqZvz3XuKKCgMgwjkOeVgdU+NqlUiruohoTNvXrXawX4WgAT6/59FMDHAVzp+ooodlhZjgdVWZZS+r0UAP6NknGDuujn9jT3qBnLwxobpXB8lPeklDBN07fn+tTUFNLpNH/vFCjLy8tYXl4O7A1jIQSKxSLf54akWq1iZGQEO3fu9HspgTfIdmkeMKWBOY6DxcVFhuUYKBQKcBwH9Xrd76UAWA3L6XQauVzO76VsGxufuK/RaKDVag29yqJpGs6cOROYm0ZxsLi4iE6n41soEEKgXC5zOykFSpDHRimlUonPmyGpVqvYvXu3L30cwoY/IfKVbduQUjIsx0DQxkepSlMYG8upi35uT3PPsGcsK5qmodlsBuamURz4OTZKYVimoAny2CilXC7zfW4Iut0uFhYWuAW7RwzL5Ct15pJhOfpUI62gnFv2c1vmoNLpNCYnJ3kR4SL1sxz2NmyeX/VeEI5caJqGs2fPckcBBYZXNwgHUSwW0Wg00Gg0/F5KpJw6dQrtdpthuUcMy+QrhuX4CFpl2c/uuG7grGV3eXXhWKlUAHDWspeC0MxP0zS02+3A3CwkMgwDqVQKExM
```

Comment:
Is this meant to be here? The last cell just has an s in it.

Reply:
Fixed